### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #643: Missing Code of conduct and Contributing documents
Hi, I see that your project don't have a Code of conduct.md and Contributing.md file
would you like me to Propose one for your team?
It will make your community profile stronger.

### PR DESCRIPTION
**Pull request by BobinMathew**
Issue [#643](https://github.com/Open-MSS/MSS/issues/643)

### CODE REVIEW

**Review by ReimarBauer — COMMENTED** [review]
looking at the source, seen there is a v2.0 
https://www.contributor-covenant.org/version/1/4/code-of-conduct.html

please use the v2.0

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (ReimarBauer) [inline CODE_OF_CONDUCT.md]: this line needs a change, 

Contributor Mission Support System Code of Conduct

**Comment by joernu76** [comment]
Personally I see the need to formalize these kind of things only after the need arises.
But the document seems pretty standard and seems reasonable.

**Review by joernu76 — DISMISSED** [review]

**Comment by BobinMathew** [comment]
what about the Pytest MSS/Test-MSS error?

**Comment by joernu76** [comment]
> what about the Pytest MSS/Test-MSS error?

We are currently setting everything up. The develop+stable branches must be fixed first, which should be completed tomorrow. Then further pull requests can be merged.

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
The latest version is https://www.contributor-covenant.org/version/2/0/code_of_conduct/
`We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, ....
`
Seems they have the kind of member states further developed and also described the enforcement guidelines and some more changes.


What are the reasons to use 1.4 and not 2.0?

**Comment by joernu76** [comment]
I agree that the V2.0 is better suited.

**Comment by BobinMathew** [comment]
Yes this https://www.contributor-covenant.org/version/2/0/code_of_conduct/ seems better

**Review by joernu76 — CHANGES_REQUESTED** [review]
Update to 2.0 required.

**Review by joernu76 — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at [INSERT CONTACT METHOD]. All complaints will be reviewed and investigated promptly and fairly.
+
```
Comment (ReimarBauer) [inline CODE_OF_CONDUCT.md:35]: we should add the email of Joern and Reimar. 

similar to

Enforcement of https://opensource.org/codeofconduct

Comment (ReimarBauer) [reply CODE_OF_CONDUCT.md:35]: - rb.proj@gmail.com
- j.ungermann@fz-juelich.de

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
Please add the names of the leader , see line 35

[INSERT CONTACT METHOD]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (ReimarBauer) [reply CODE_OF_CONDUCT.md:35]: this little bit I add afterwards

**Review by ReimarBauer — APPROVED** [review]

**Pull request merged**